src/dos/dos_keyboard_layout.cpp

```diff
@@ -1322,9 +1323,59 @@ void DOS_KeyboardLayout_ShutDown(Section* /*sec*/) {
 	delete test;	
 }
 
+std::map<std::string, int> country_code_map {
+    {"cz243", 42},
+    {"dk", 45},
+    {"gr", 49},
+    {"gk", 30},
+    {"sp", 34},
+    {"su", 358},
+    {"fr", 33},
+    {"hu", 36},
+    {"hu208", 36},
+    {"is161", 354},
+    {"it", 39},
+    {"nl", 31},
+    {"no", 47},
+    {"pl", 48},
+    {"br", 55},
+    {"ru", 7},
+    {"hr", 38},
+    {"sk", 42},
+    {"sv", 46},
+    {"tr", 90},
+    {"ur", 380},
+    {"bl", 375},
+    {"si", 386},
+    {"et", 372},
+    {"sg", 41},
+    {"po", 351},
+    {"sf", 41},
```

Comment:
std::map does help to connect the short-hand country strings to the country numbers. 

But the numbers themselves are still used as magic numbers elsewhere.
For example, 41 is used:

![2022-01-04_06-41](https://user-images.githubusercontent.com/1557255/148076024-4eb98ce0-94ef-42ef-baf5-8ef5e96b4a51.png)

and

![2022-01-04_06-41_1](https://user-images.githubusercontent.com/1557255/148076042-216c6a44-7ff7-4e7a-909d-ac488971cc5b.png)

So we still need an enum to state a human-readable country name for whatever 41 represents, then use that instead of 41 everywhere.

Reply:
Done.

Reply:
In this comment:

![2022-01-12_11-10](https://user-images.githubusercontent.com/1557255/149206016-cbbf8867-421f-4078-8d9d-54791b42bd93.png)

I was hoping the country enum values could be used here too.